public/js/p3/widget/WorkspaceObjectSelector.js

```diff
@@ -556,12 +563,28 @@ define([
 				this.refreshWorkspaceItems();
 			}
 			Topic.subscribe("/refreshWorkspace", lang.hitch(this, "refreshWorkspaceItems"));
+			Topic.subscribe("/upload", lang.hitch(this, "onUploadMessage"));
 			this.searchBox.set('disabled', this.disabled);
 			this.searchBox.set('required', this.required);
 			this.searchBox.set('placeHolder', this.placeHolder);
 			this.searchBox.labelFunc = this.labelFunc;
 		},
 
+		onUploadMessage(msg){
```

Comment:
This needs to be like below,
```
onUploadMessage: function(msg){
```

Reply:
will make this change, thanks for pointing it out.